src/renderer/containers/Friend/FriendOverview/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, {FC, ReactNode, useEffect, useState} from 'react';
+import {useSelector} from 'react-redux';
+import {useParams} from 'react-router-dom';
+import axios from 'axios';
+
+import DetailPanel from '@renderer/components/DetailPanel';
+import Qr from '@renderer/components/Qr';
+import {getActivePrimaryValidatorConfig, getManagedFriends} from '@renderer/selectors';
+import {formatAddress} from '@renderer/utils/address';
+
+import './FriendOverview.scss';
+
+const FriendOverview: FC = () => {
+  const {friendNumber} = useParams();
+  const [balance, setBalance] = useState<number | null>(null);
+  const [loading, setLoading] = useState<boolean>(true);
+  const activePrimaryValidator = useSelector(getActivePrimaryValidatorConfig);
+  const managedFriends = useSelector(getManagedFriends);
+  const managedFriend = managedFriends[friendNumber];
+
+  useEffect(() => {
+    if (!activePrimaryValidator) return;
+
+    const fetchData = async (): Promise<void> => {
+      const {ip_address: ipAddress, port, protocol} = activePrimaryValidator;
+      const address = formatAddress(ipAddress, port, protocol);
+
+      setLoading(true);
+      const {data} = await axios.get(`${address}/friend/${friendNumber}/balance`);
```

Comment:
should be `${address}/accounts/${accountNumber}/balance` per our discussion on slack